src/scss/templates/_blog.scss

```diff
@@ -160,3 +160,26 @@
 	}
 
 }
+
+.how-to-wrapper {
+	picture svg,
+	figure svg {
```

Comment:
probably only svg as selector here is enough

Reply:
it cannot, there is other svg for the carousel arrow @foria